utilities/transactions/write_prepared_transaction_test.cc

```diff
@@ -283,6 +287,22 @@ class WritePreparedTransactionTest : public TransactionTest {
     }
     rocksdb::SyncPoint::GetInstance()->DisableProcessing();
   }
+
+  void VerifyInternalKeys(const std::vector<KeyVersion>& expected_versions) {
+    std::vector<KeyVersion> versions;
+    ASSERT_OK(GetAllKeyVersions(db, expected_versions.front().user_key,
+                                expected_versions.back().user_key, &versions));
+    ASSERT_EQ(expected_versions.size(), versions.size());
+    for (size_t i = 0; i < versions.size(); i++) {
+      ASSERT_EQ(expected_versions[i].user_key, versions[i].user_key);
+      ASSERT_EQ(expected_versions[i].sequence, versions[i].sequence);
+      ASSERT_EQ(expected_versions[i].type, versions[i].type);
+      if (versions[i].type != kTypeDeletion &&
+          versions[i].type != kTypeSingleDeletion) {
```

Comment:
How about Range Delete?

Reply:
RangeDelete is not supported for transactions (seems even for StackableDB).

Reply:
Can you assert that here? Just in case it changes in future.